coverage-targets.js

```diff
@@ -6,10 +6,10 @@
 // subset of files to check against these targets.
 module.exports = {
   global: {
-    lines: 64.39,
-    branches: 53.03,
-    statements: 63.66,
-    functions: 56.67,
+    lines: 68.63,
+    branches: 47.48,
+    statements: 66.9,
+    functions: 62.48,
   },
```

Comment:
Was getting `Error: Coverage EXCEEDS threshold for global and must be bumped`

![Screenshot 2023-03-03 at 11 29 33 AM](https://user-images.githubusercontent.com/8112138/222809633-6e7f7749-c8da-4339-ae86-b93d68b1733a.png)